recipes/awsbatch_config.rb

```diff
@@ -64,3 +64,8 @@
     command "#{node['cfncluster']['awsbatch_virtualenv_path']}/bin/pip install aws-parallelcluster==#{node['cfncluster']['cfncluster-version']}"
   end
 end
+
+if node['cfncluster']['dcv_enabled'] == "master"
+  # Activate DCV on head node
+  include_recipe 'aws-parallelcluster::dcv_config'
```

Comment:
isn't the `head_node_base_config` recipe taking care of calling `dcv_config`?

Reply:
Ah yes, so it was actually an issue of DCV not enabled in kitchen test. PR is updated